Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -53,12 +51,8 @@ protected override IHost CreateHost(IHostBuilder builder)
         return base.CreateHost(builder);
     }
 
-    private static void ConfigureContainer(ContainerBuilder builder)
-    {
+    private static void ConfigureContainer(ContainerBuilder builder) =>
+
         // in the api tests we need to replace current user with TestUser
         builder.RegisterModule(new TestModule(true)); // this allows certain components to be overriden
-
-        // Api does not depend on migrations assembly, tests are
-        builder.RegisterModule(new EntityFrameworkModule { RegisterMigrationsAssembly = true });
```

Comment:
This re-registration of EntifyFrameworkModule was needed only to pass RegisterMigrationsAssembly flag set to true. This flag is now being passed through the TestConfiguration.cs